source/2015-03-25-ten-tools-to-help-build-automated-marketing-websites.html.markdown

```diff
@@ -0,0 +1,123 @@
+---
+title: Ten tools to help build automated marketing websites.
+subtitle: Marketing automation tools for the rest of us.
+date: 2015-03-25 06:06 +0000
+tags: seo, sem, automated marketing
+category: marketing
+author:
```

Comment:
I think this metadata should live in a single place. Maybe it can be a data.yml file that we import and we just refer to it as `author: j_mcnally` and it pulls the rest of the data.